string warden_list_req
---
string[] warden_list

